.gitignore

```
@@ -58,3 +58,6 @@ Makefile.local
 
 # scan-build artifacts
 scan-build/
+
+# compile_and_test_for_boards default "results" directory
+results/*
```

This would make the directory also ignored if it is in sub directories.

```suggestion
results/
```

---

Addressed and directly squashed